### ISSUE TITLE
init_wallet method does not extists.

### ISSUE THREAD

**Issue by miratcan** [blocktrail/blocktrail-sdk-python#8]
There must be init_wallet method am I right?

![screen shot 2016-03-11 at 19 45 18](https://cloud.githubusercontent.com/assets/140968/13710378/d26460b4-e7c1-11e5-9b35-9c632ccb3d66.png)

**Comment by YiumPotato** [comment]
Yup. Was hyped to continue my python project till I noticed there isn't that method. :( I wish the python repo was better maintained like the node.js or php one. If I notice it's just a simple bug I'll do a pull request

**Comment by gugadev** [comment]
@rubensayshi So, any update about this?
